addon/initialize.js

```diff
@@ -12,11 +12,13 @@ function unwrap(input) {
 }
 
 export default function() {
-  ValidatorsMessages.reopen({
+  ValidatorsMessages.reopenClass({
     i18n: Ember.inject.service(),
     prefix: 'errors',
```

Comment:
Moving this to the constructor will require a doc change to https://github.com/jasonmit/ember-i18n-cp-validations#customizing-the-prefix